pkg/operation/terraformer/config.go

```diff
@@ -100,6 +97,12 @@ func (t *Terraformer) prepare() (int, error) {
 	return numberOfExistingResources, nil
 }
 
+// ConfigExists returns true if all three Terraform configuration secrets/configmaps exist, and false otherwise.
+func (t *Terraformer) ConfigExists() (bool, error) {
+	numberOfExistingResources, err := t.prepare()
+	return numberOfExistingResources == 3, err
```

Comment:
Would make it a constant -> Can be reused [here](https://github.com/gardener/gardener/blob/master/pkg/operation/terraformer/terraformer.go#L116)

Reply:
done, PTAL